cartography/sync.py

```diff
@@ -106,6 +106,7 @@ def run_with_config(sync, config):
         neo4j_driver = GraphDatabase.driver(
             config.neo4j_uri,
             auth=neo4j_auth,
+            max_connection_lifetime=config.neo4j_max_connection_lifetime,
```

Comment:
the default for `config.neo4j_max_connection_lifetime` is none - if `max_connection_lifetime` gets set to none is that alright? would it be better to explicitly set this to 3600 in CLI and config? that would make the docstring update in config.py make more sense to users as well

Reply:
True, yeah will do.

Reply:
Done, see cli.py.